GUI/Main/Main.cs

```diff
@@ -765,17 +767,45 @@ public void LaunchGame()
             var binary = split[0];
             var args = string.Join(" ", split.Skip(1));
 
+            gameTime = new Stopwatch();
+
+            TimeLog t = new TimeLog(CurrentInstance.CkanDir());
```

Comment:
This object isn't being used (since we don't need to touch the file till after the game exits). Can it be removed?

Reply:
That was another thing that I just needed for testing, will remove